src/platform/plugins/shared/discover/public/context_awareness/profile_providers/observability/traces_data_source_profile/accessors/get_columns_configuration.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import { UnifiedDataTableSummaryColumnHeader } from '@kbn/unified-data-table';
+import type { CustomGridColumnProps } from '@kbn/unified-data-table';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { DataSourceProfileProvider } from '../../../..';
+
+export const getColumnsConfiguration: DataSourceProfileProvider['profile']['getColumnsConfiguration'] =
+  (prev) => () => ({
+    ...(prev ? prev() : {}),
+    _source: ({ column, headerRowHeight }: CustomGridColumnProps) => ({
+      ...column,
+      display: (
+        <DataTableSummaryColumnHeaderTracesContext
+          columnDisplayName={column.displayAsText ?? 'Summary'}
+          headerRowHeight={headerRowHeight}
+        />
+      ),
+    }),
+  });
+
+export const DataTableSummaryColumnHeaderTracesContext = ({
+  headerRowHeight = 1,
+  columnDisplayName,
+}: {
+  headerRowHeight?: number;
+  columnDisplayName: string;
+}) => {
+  const tooltipTitle = i18n.translate(
+    'discover.unifiedDataTable.tableHeader.tracesContext.sourceFieldIconTooltipTitle',
+    {
+      defaultMessage: 'Summary including the following fields',
+    }
+  );
+  const tooltipContent = (
+    <FormattedMessage
+      id="discover.unifiedDataTable.tableHeader.tracesContext.sourceFieldIconTooltip"
+      defaultMessage="Displays the most relevant resource identifiers like:{br}{br}service.name{br}event.outcome{br}transaction.name{br}etc."
```

Comment:
```suggestion
      defaultMessage="Displays the most relevant resource identifiers like:{br}{br}service.name{br}event.outcome{br}span.name{br}etc."
```

We're showing both the span and transaction names, and using span instead of transaction makes it a bit more aligned with OTel.

Reply:
@iblancof The fields were already discussed with UX, Copy and Product team [here](https://github.com/elastic/logs-dev/issues/187#issuecomment-3008489756). 

Can you explain what exactly do you mean aligned more with Otel ?

Reply:
Oh, sorry, I didn’t know about that earlier discussion!

To give a bit of context, we have trace documents ingested either through APM or directly from OTel.

Maybe my point isn’t that important since `event.outcome` isn’t part of the OTel schema either when it’s unprocessed (not processed by the `elastictraceprocessor`). 

When I said “more aligned with OTel,” I meant that unprocessed OTel data doesn’t have `transaction.name` (or `event.outcome`), but it always has `span.name`. In APM, sometimes we get `span.name` and sometimes `transaction.name`. 

But It’s just a small detail, so I don’t see it as a problem. I might be making things more complicated than needed, so let’s just leave it as is, I don’t want to confuse anyone!